model_finetuning/download_reddit_finetuning_data.py

```diff
@@ -56,6 +63,30 @@ def clean_text(text):
 	return text
 
 
+def get_with_retry(link: str):
+	"""
+	Performs a request using a re-try mechanism. Throws if
+	"""
+	# This could be a configuration within a dataset ini maybe?
```

Comment:
I think 99% of users won't need to adjust these parameters so I think it's better to 'kiss' and leave this all as it is.